frontend/src/state/rootEpic.js

```diff
@@ -0,0 +1,10 @@
+import { combineEpics } from 'redux-observable';
+
+import userEpics from './user/userEpics';
+
+// Combine all epics into one; this is the expected behaviour of redux-observable.
+const rootEpic = combineEpics(
+    userEpics,
```

Comment:
Not a requested change, just a suggestion. Should we add global error handling to catch anything that manages to fall through the other epics uncaught? 

Example near the bottom of the page here:

https://redux-observable.js.org/docs/basics/SettingUpTheMiddleware.html